docs/modules/ROOT/pages/how-to-guides/proc_managed_services_onboarding.adoc

```diff
@@ -3,101 +3,72 @@ When an application is ready for release, the Development team contacts the Mana
 
 The Managed Environment team creates the following objects on the Managed workspace:
 
-* *ReleaseStrategy* - The `ReleaseStrategy` object, created by the Managed Environment team, specifies the release pipeline that the system uses to release an application in their environment, as well as which enterprise contract the system must first validate.
-
-* *ReleasePlanAdmission* - The Managed Environment team creates the ReleasePlanAdmission object in response to the `ReleasePlan` object created by the development team. It indicates that the Managed Environment team has approved the xref:how-to-guides/proc_release_application.adoc#_creating_a_releaseplan_object[`ReleasePlan`] object for an application. It also specifies a `ReleaseStrategy` to be used for that particular Application and, optionally, an Environment to which the application will be deployed.
+* *ReleasePlanAdmission* - The Managed Environment team creates the `ReleasePlanAdmission` object in response to the `ReleasePlan` object created by the development team. It indicates that the Managed Environment team has approved the link:https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/how-to-guides/proc_release_plan/[ReleasePlan] object for an application.
```

Comment:
The `ReleasePlanAdmission` could already exist. The managed team creates or updates the RPA to include the application specified by the development team. So, the Managed Environment team approves the application specified in the ReleasePlan object.